src/metricscollector/config/config.go

```diff
@@ -54,12 +55,21 @@ var defaultCollectorConfig = CollectorConfig{
 	PollInterval:    DefaultPollInterval,
 }
 
+type DataPruneConfig struct {
+	CutoffDays int `yaml:"cutoff_days"`
+}
+
+var defaultDataPruneConig = DataPruneConfig{
```

Comment:
Yeah, those were not intended names, just to put across our understanding for clarity to depict their jobs used those names:)

So we will go ahead with this structure which is more clear. We did struggle with the previous approach.